stdlib/public/core/Integers.swift.gyb

```diff
@@ -2555,11 +2557,8 @@ extension FixedWidthInteger {
 
 % for x in binaryArithmetic['Numeric'] + binaryArithmetic["BinaryInteger"][:1]:
 %   callLabel = x.firstArg + ': ' if not x.firstArg == '_' else ''
-// FIXME(integers): pending optimizer work on handling the case where the
-// boolean value is wrapped into a two-case enum and then immediately
```

Comment:
This seems to be a reference to the original `ArithmeticOverflow` enum that was mercifully removed. The block remains disabled because the holdup is with overload resolution, I think, and nothing to do with optimization of two-case enums.